server/src/test/java/io/druid/client/CachingClusteredClientTest.java

```diff
@@ -205,7 +205,7 @@
       )
   );
   private static final List<AggregatorFactory> RENAMED_AGGS = Arrays.asList(
-      new CountAggregatorFactory("rows2"),
+      new CountAggregatorFactory("rows"),
```

Comment:
This was not for a better test, but to make unit tests passed.

```CachingClusteredClientTest.testTopN*``` tests check the cached result, and here the same post aggregator should be used except their names. So, I just changed test queries to use ```POST_AGG``` which is also used for caching like [here](https://github.com/druid-io/druid/pull/3899/files/ddb4eee8859b1bd5d433b2df92e742e2a2ee9dc0#diff-7254bc82bb1d7b6afc78e03268c96806R821). 

To make ```POST_AGG``` work properly, the ```FieldAccessPostAggregator.fieldName``` should be the one specified in ```AGGS```. So, I changed the name of ```CountAggregatorFactory``` from ```rows2``` to ```rows```.

Since there is another renamed field ```impers2``` in ```RENAMED_AGG```, I thought it would not be a problem in other tests for group-by with renaming. If not, please let me know.

The change in makeTopNResultsWithoutRename() is just for my convenience. If its name is verbose, I'll change it.